.cbmc-batch/jobs/aws_priority_queue_init_dynamic/aws_priority_queue_init_dynamic_harness.c

```diff
@@ -22,31 +22,31 @@
 void aws_priority_queue_init_dynamic_harness() {
 
     /* data structure */
-    struct aws_priority_queue *queue;
+    struct aws_priority_queue queue;
 
     /* parameters */
-    struct aws_allocator *allocator;
+    struct aws_allocator *allocator = can_fail_allocator();
     size_t item_size;
     size_t initial_item_allocation;
     size_t len;
 
     /* assumptions */
-    ASSUME_VALID_MEMORY(queue);
-    ASSUME_CAN_FAIL_ALLOCATOR(allocator);
+    __CPROVER_assume(&queue != NULL);
+    __CPROVER_assume(allocator != NULL);
```

Comment:
Why do you use `can_fail_allocator` and then make this assumption? Wouldn't be easier just set an allocator that can't fail?